Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.7-slim
+
+COPY requirements.txt /
+RUN buildDeps='build-essential gcc gfortran python3-dev libpq-dev' \
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5942194)